LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/Definitions/IAttestation.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp.Definitions
+{
+    internal interface IAttestation
```

Comment:
Is there a reason for this being `internal` and other types being `public` in this namespace?

Also, this abstraction isn't used anywhere except on its one implementation `SymmetricKeyAttestation`. Should we remove it in case of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)?

Reply:
Interface removed